### PR TITLE
Add Open Source Transition Resources repository to readme

### PR DESCRIPTION
**Pull request by sarahcode2112**
Add repositories to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +43,10 @@ Click on the [CONTRIBUTING.md](https://github.com/amandamartin-dev/women-led-ope
 
 [Njong Emy](https://github.com/Njong392) - [Abbreve](https://github.com/Njong392/Abbreve)
 
+[Red Hat Pride](https://github.com/RedHatPride) - [Open Source Transition Resources](https://github.com/RedHatPride/open-source-transition-resources)
```
Comment (CBID2) [inline README.md]:
```suggestion
- [Red Hat Pride](https://github.com/RedHatPride) 
[Open Source Transition Resources](https://github.com/RedHatPride/open-source-transition-resources)
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @sarahcode2112. Thanks for contributing! :) Just made a change to your PR

File: README.md (hunk 1 above)
Comment (sarahcode2112) [reply README.md]: Looks great, thank you! :)

**Review by CBID2 — APPROVED** [review]
Well done

**Pull request merged**